# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linuxwacom/xorg-7.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- linuxwacom-0.7.2_orig/acinclude.m4	2005-12-22 00:04:36.000000000 +0100
+++ linuxwacom-0.7.2/acinclude.m4	2006-02-05 14:06:38.219238250 +0100
@@ -220,7 +220,7 @@
 
 	if test -n "$WCM_XORGSDK"; then
 		AC_MSG_CHECKING(for valid Xorg SDK)
-		if test -f $WCM_XORGSDK/include/xf86Version.h; then
+		if test -f $WCM_XORGSDK/include/xorg/xf86Version.h; then
 			WCM_ENV_XORGSDK=yes
 			AC_MSG_RESULT(ok)
 		elif test -f $WCM_XORGSDK/xc/include/xf86Version.h; then
--- linuxwacom-0.7.2_orig/src/Makefile.am	2005-10-26 02:36:47.000000000 +0200
+++ linuxwacom-0.7.2/src/Makefile.am	2006-02-05 14:12:41.985972250 +0100
@@ -52,7 +52,8 @@
 XSERVER64	=@WCM_XSERVER64@
 
 if WCM_ENV_XORGSDK
-DRIVER_INCLUDES = -I$(XORGSDK_DIR)/include
+DRIVER_INCLUDES = -I$(XORGSDK_DIR)/include -I$(XORGSDK_DIR)/include/xorg \
+	-I$(XORGSDK_DIR)/include/X11
 endif
 
 if WCM_ENV_XF86
--- linuxwacom-0.7.2/src/Makefile.am_orig	2006-02-05 15:40:28.000000000 +0100
+++ linuxwacom-0.7.2/src/Makefile.am	2006-02-05 15:41:01.000000000 +0100
@@ -7,7 +7,7 @@
 wacomcfgdir = $(includedir)/wacomcfg
 wacomcfg_HEADERS = wacomcfg.h
 
-x86moduledir = @WCM_XLIBDIR@/modules/input
+x86moduledir = @WCM_XLIBDIR@/xorg/modules/input
 x86module_DATA = @WCM_XF86MODULES@
 
 AM_CFLAGS = -Wall -pedantic
